hack/smoketest/main.go

```diff
@@ -64,8 +64,15 @@ func main() {
 		ApplyManifest(filepath.Join("install", "crds.yaml")),
 		ApplyManifest(filepath.Join("install", "operator.yaml")),
 		WaitForDeploymentAvailable("cockroach-operator"),
-		WaitForSecret("cockroach-operator-webhook-tls"),
-		ApplyManifest(filepath.Join("examples", "example.yaml")),
+		WaitForSecret("cockroach-operator-webhook-ca"),
+		sleep(10 * time.Second), // just give the manager time to write the TLS certs to disk
+		ApplyManifestWithReplacements(
+			filepath.Join("examples", "example.yaml"),
+			map[string]string{
+				// 2Gi was preventing the pods from getting scheduled in GitHub actions
+				"memory: 2Gi": "memory: 1Gi",
```

Comment:
should we just update example.yaml to be 1Gi? i feel i always replace it with 1Gi manually too